geometry_msgs/Vector3 relative_position

float32 steering_angle
float32 rotate_angle
string name